wordpress-mobile/WordPress-Android
# Issue/remove bp onboarding dialog

This PR removes a temporary Blogging Prompts onboarding dialog on app start.

To test:
- Perform a fresh installation of vanilla JP app.
- Login and navigate to main screen. Dismiss quick start prompt.
- Kill the app and reopen it.
- Confirm that onboarding dialog is not displayed.


## Regression Notes
1. Potential unintended areas of impact
Something is mysteriously broken.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
